src/common/meta/src/sequence.rs

```diff
@@ -164,15 +164,15 @@ impl Inner {
 
             let req = CompareAndPutRequest {
                 key: key.to_vec(),
-                expect,
+                expect: expect.clone(),
```

Comment:
This seems to introduce an extra clone?

Reply:
I don't know why it didn't need before

Reply:
This is because it's a retry loop. If the request failed, we always update `expect` so NLL analysis should know here is no need for cloning.

If it can't compile, either the `clone_from` clippy has an edge case, or it's a compiler regression.

Reply:
Resolve in 9a8bcac12b6f1e68c899ddcf88187d216262228e.